app/views/templates/report-details.html.slim

```diff
@@ -86,7 +86,8 @@
       .panel id='{{ testAnchor(result) }}' ng-class='{"panel-success": result.passed && result.active, "panel-danger": !result.passed && result.active, "panel-warning": !result.active}'
         .panel-body
           test-key-label.pull-right ng-if='result.key' key='result.key'
-          h4 ng-class='{"text-success": result.passed && result.active, "text-danger": !result.passed && result.active, "text-warning": !result.active}' {{ result.name }}
+          h4 ng-class='{"text-success": result.passed && result.active, "text-danger": !result.passed && result.active, "text-warning": !result.active}'
+            a href='#' ui-sref='org.tests.show({ testId: result.testId, orgId: result.project.organizationId })' {{ result.name }}
```

Comment:
`ui-sref='org.tests.show({ testId: result.testId })'`

Does this work without the `orgName` parameter? I assume it's because it keeps the same value by default, because the parameter is required for a child of the `org.` state.